challenges/lib/src/main/java/challenges/Library.java

```diff
@@ -4,7 +4,31 @@
 package challenges;
 
 public class Library {
-    public boolean someLibraryMethod() {
-        return true;
+    public static int binarySearch(int[] arr, int target){
+      //if empty return -1
```

Comment:
Nice, always check for the base case. No point in doing all this work if our given input was empty.